phpunit/navigation-page-test.php

```diff
@@ -83,7 +83,7 @@ public function test_gutenberg_navigation_editor_preload_menus_initializes_creat
 		$this->callback
 			->expects( $this->once() )
 			->method( 'preload_menus_rest_pre_dispatch_callback' )
-			->willReturn( new $response );
+			->willReturn( new $response() );
```

Comment:
I think the new keyword was added here by mistake (by another developer).
```suggestion
			->willReturn( $response );
```

Reply:
Nice catch! I agree this looks like a mistake.

I wonder why the test doesn't error out on it. I suggest this test needs a careful review.

Reply:
I've reviewed this test method and IMO it is fine.
It looks like it's just a typo.